src/compo/mopitt_co_nc2ioda.py

```diff
@@ -158,11 +158,11 @@ def _read(self):
                 self.outdata[('datetime', 'MetaData')] = times[flg]
                 self.outdata[('latitude', 'MetaData')] = lats[flg]
                 self.outdata[('longitude', 'MetaData')] = lons[flg]
-                self.outdata[('apriori_term', 'MetaData')] = ap_tc[flg]
+                self.outdata[('apriori_term', 'Smoothing')] = ap_tc[flg]
```

Comment:
Said this in the UFO PR, not a huge fan of `Smoothing` (is this added to the IODA conventions?). What about something more like `Retrieval`? But, I don't feel strong enough to derail this/associated PRs.

Reply:
I'm kinda with you on this. Something with `Retrieval` in it like `RetrievalAncillaryData` or something, but that just seems bloated and awkward. I don't have a good alternative either. Also, don't feel strongly enough to derail things.

Reply:
See comment response here: https://github.com/JCSDA-internal/ufo/pull/2183#discussion_r937136336
I like Bryan's second suggestion which is the most general for all the data we want to put in this group. I could change it to `RtrvlAncData` ?

Reply:
Sure, works for me.

Reply:
I changed the name of the group to something less awkward now.